src/main/java/com/sprints/controllers/SprintsController.java

```diff
@@ -48,8 +53,18 @@ public SprintDomain findSprintById(@PathVariable("id") String id) throws EntityN
 	
 	@ApiOperation(value = "Find all sprints")
 	@RequestMapping(method=RequestMethod.GET)
-	public List<SprintDomain> findAllSprints(){
-		return sprintsService.findAll();
+	@DisallowUndeclaredRequestParams
+	public List<SprintDomain> findAllSprints( @RequestParam("name") Optional<String> name,
+												@RequestParam("technology") Optional<String> technology,
+												@RequestParam("start_date") @DateTimeFormat(iso = ISO.DATE) Optional<LocalDate> start_date,
+												@RequestParam("end_date") @DateTimeFormat(iso = ISO.DATE) Optional<LocalDate> end_date){
+		
+		if(name.isEmpty() && technology.isEmpty() && start_date.isEmpty() && end_date.isEmpty()) {
```

Comment:
We well need take this if to serviceImpl

Reply:
Fixed